plugin.video.mediathekview/resources/lib/mvutils.py

```diff
@@ -0,0 +1,313 @@
+# -*- coding: utf-8 -*-
+"""
+Utilities module
+
+Copyright (c) 2017-2019, Leo Moll
+Licensed under MIT License
+"""
+
+from __future__ import unicode_literals
+
+import os
+import sys
+import stat
+import string
+
+# pylint: disable=import-error
+try:
+    # Python 3.x
+    from urllib.parse import urlencode
+    from urllib.request import urlopen
+except ImportError:
+    from urllib import urlencode
+    from urllib2 import urlopen
+
+from contextlib import closing
+from resources.lib.exceptions import ExitRequested
+
+# -- Kodi Specific Imports ----------------------------------
+try:
+    import xbmcvfs
+    IS_KODI = True
+except ImportError:
+    IS_KODI = False
+
```

Comment:
Looking at this file makes me wish python2 had died sooner :)

Reply:
Me too